jerry-core/parser/js/js-parser-statm.c

```diff
@@ -1558,6 +1558,14 @@ parser_parse_statements (parser_context_t *context_p) /**< context */
   parser_stack_push_uint8 (context_p, PARSER_STATEMENT_START);
   parser_stack_iterator_init (context_p, &context_p->last_statement);
 
+#ifdef DEBUGGER
```

Comment:
DEBUGGER should be JERRY_DEBUGGER, since we need to prefix all defines with JERRY_